Redo v16.3 GSI script updates to scripts/exglobal_atmos_analysis.sh

KateFriedman-NOAA opened the pull request:
**Description**

This PR brings in GFSv16.3 GSI script changes that were previously reverted from PR #1158 in PR #1238 while dealing with a CRTM issue. That issue has now been resolved, so these updates need to come back in to resolve an issue with the GSI assimilating too low of a number of observations.

Resolves #1494
Resolves #1550 

**Type of change**

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Ops update

**How Has This Been Tested?**

- [x] Clone and Build tests on Orion
- [x] Cycled test on Orion - examined by @emilyhcliu (see comments in #1494)
  
**Checklist**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [x] Any dependent changes have been merged and published

KateFriedman-NOAA committed:
Redo v16.3 GSI script updates to scripts/exglobal_atmos_analysis.sh

- These updates to scripts/exglobal_atmos_analysis.sh are being put in
again after a CRTM issue was resolved.
- These updates are needed to resolve an issue with the GSI assimilating
too low of a number of observations.
- These updates are from the GFSv16.3 operational upgrade.

Refs #1494

KateFriedman-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/v16.3_gsi_script_updates

KateFriedman-NOAA committed:
Update nsig_ext value in config.anal

- Change nsig_ext value from 56 to 45 within SETUP variable setting.

Refs #1494

KateFriedman-NOAA committed:
Update INCREMENTS_TO_ZERO and INCVARS_ZERO_STRAT in config.base.emc.dyn

Refs #1494

KateFriedman-NOAA committed:
Further settings updates in scripts/exglobal_atmos_analysis.sh

- Update nsig_ext from 12 to 45
- Update dmesh(4) from 25.0 to 50.0

Refs #1494

WalterKolczynski-NOAA commented on parm/config/config.base.emc.dyn:
```diff
@@ -354,13 +354,13 @@ export MAKE_NSSTBUFR="@MAKE_NSSTBUFR@"
 
 # Analysis increments to zero in CALCINCEXEC
-export INCREMENTS_TO_ZERO="'liq_wat_inc','icmr_inc'"
+export INCREMENTS_TO_ZERO="'liq_wat_inc','icmr_inc','rwmr_inc','snmr_inc','grle_inc'"
```
If this is a production update, shouldn't these changes also be made to `config.base.nco.static`?

KateFriedman-NOAA replied:
Yes, good catch. I have now also updated `config.base.nco.static` for the INCREMENTS variables @ 79d08613. Even though `config.base.nco.static` needs a full review and possible redo it's worth keeping it up-to-date with these changes. Thanks @WalterKolczynski-NOAA !

KateFriedman-NOAA committed:
Also update config.base.nco.static for v16.3 settings

- Update INCREMENTS_TO_ZERO and INCVARS_ZERO_STRAT similarly
to config.base.emc.dyn

Refs #1494

KateFriedman-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/v16.3_gsi_script_updates

KateFriedman-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/v16.3_gsi_script_updates

KateFriedman-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/v16.3_gsi_script_updates

KateFriedman-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/v16.3_gsi_script_updates

KateFriedman-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/v16.3_gsi_script_updates

KateFriedman-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/v16.3_gsi_script_updates

KateFriedman-NOAA committed:
Update gsi_ver to 20230112 in fix.ver

Installed updated GSI-fix set on supported platforms under
the 20230112 timestamp.

KateFriedman-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/v16.3_gsi_script_updates
